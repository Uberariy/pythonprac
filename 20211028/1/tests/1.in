print("ЫЫЫЫ", list(genfib(2, 4)))