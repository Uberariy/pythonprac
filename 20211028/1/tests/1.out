ЫЫЫЫ [2, 3, 5]
